pr_agent/tools/pr_reviewer.py

```diff
@@ -219,7 +225,7 @@ def _prepare_pr_review(self) -> str:
             logging.info(f"Markdown response:\n{markdown_text}")
 
         if markdown_text == None or len(markdown_text) == 0:
-            markdown_text = review
+            markdown_text = ""
```

Comment:
**Suggestion:** Instead of checking if `markdown_text` is `None` or its length is 0, you can simply use `if not markdown_text:` to check if the string is empty or `None`.
```suggestion
        if not markdown_text:
            markdown_text = ""
```